src/extract.rs

```diff
@@ -11,13 +11,47 @@ use futures_core::ready;
 
 use crate::{dev::Payload, Error, HttpRequest};
 
-/// Trait implemented by types that can be extracted from request.
+/// [`FromRequest`] is one of the most commonly used trait, although some may not realize it.
```

Comment:
IMHO this line is a bit bogus since it doesn't provide a user with any meaningful information. 
I'd suggest removing it completely.